### PR TITLE
Refactor the roles, replace glusterfs[_.]infra with gluster[._]infra

### PR DESCRIPTION
**Pull request by sac**

### CODE REVIEW

**Comment by sac** [comment]
@devyanikota can you please review and merge this?

**Comment by sac** [comment]
@devyanikota can you please review and merge this?

File: README.md [hunk 1]
```diff
@@ -29,51 +29,51 @@ respective sub-roles directory.
...
+| gluster_infra_lv_thinpoolname || glusterfs_thinpool | Optional variable. If omitted glusterfs_thinpool is used for thinpoolname. |
+| gluster_infra_lv_thinpoolsize || 100%FREE | Thinpool size, if not set, entire disk is used. Include the unit [G\|M\|K] |
+| gluster_infra_lv_logicalvols || UNDEF | This is a list of hash/dictionary variables, with keys, lvname and lvsize. See below for example. |
+| gluster_infra_lv_thicklvname || gluster_infra_lv_thicklvname | Optional. Needed only if thick volume has to be created. The variable will have default name gluster_infra_lv_thicklvname if thicklvsize is defined. |
+| gluster_infra_lv_thicklvsize || UNDEF | Optional. Needed only if thick volume has to be created. Include the unit [G\|M\|K]|
+| gluster_infra_mount_devices | | UNDEF | This is a dictionary with mount values. path, and lv are the keys. |
+| gluster_infra_ssd_disk | | UNDEF | SSD disk for cache setup, specific to HC setups. Should be absolute path. e.g /dev/sdc |
+| gluster_infra_lv_cachelvname | | glusterfs_ssd_cache | Optional varialbe, if omitted glusterfs_ssd_cache is used by default. |
```
Comment (devyanikota) [inline README.md]: nit: s / varialbe / variable

File: roles/backend_setup/README.md [hunk 2]
```diff
@@ -21,39 +21,39 @@ Role Variables
...
+| gluster_infra_lv_thinpoolname || glusterfs_thinpool | Optional variable. If omitted glusterfs_thinpool is used for thinpoolname. |
+| gluster_infra_lv_thinpoolsize || 100%FREE | Thinpool size, if not set, entire disk is used. Include the unit [G\|M\|K] |
+| gluster_infra_lv_logicalvols || UNDEF | This is a list of hash/dictionary variables, with keys, lvname and lvsize. See below for example. |
+| gluster_infra_lv_thicklvname || gluster_infra_lv_thicklvname | Optional. Needed only if thick volume has to be created. The variable will have default name gluster_infra_lv_thicklvname if thicklvsize is defined. |
+| gluster_infra_lv_thicklvsize || UNDEF | Optional. Needed only if thick volume has to be created. Include the unit [G\|M\|K]|
+| gluster_infra_mount_devices | | UNDEF | This is a dictionary with mount values. path, and lv are the keys. |
+| gluster_infra_ssd_disk | | UNDEF | SSD disk for cache setup, specific to HC setups. Should be absolute path. e.g /dev/sdc |
+| gluster_infra_lv_cachelvname | | glusterfs_ssd_cache | Optional varialbe, if omitted glusterfs_ssd_cache is used by default. |
```
Comment (devyanikota) [inline roles/backend_setup/README.md]: nit: s / varialbe / variable

**Comment by sac** [comment]
Ack! Fixed typo

**Pull request merged**